File: FutsalSystemFront/src/app/Models/AnnouncementDto.ts
export class AnnouncementDto {
    public Id: number;
    public Title: string;
    public CreationDate: string;
    public Message: string;

    constructor(id: number, title: string, creationDate: string, message: string) {
        this.Id = id;
        this.Title = title;
        this.CreationDate = creationDate;
        this.Message = message;
    }
}

File: FutsalSystem/FutsalSystem/Models/DTO/Announcement/AnnouncementDTO.cs
using FutsalSystem.Base.Interface;

namespace FutsalSystem.Models.DTO.Announcement
{
    public class AnnouncementDTO : IBaseEntity
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string CreationDate { get; set; }
        public string Message { get; set; }
    }
}
